js/augurs-prophet-js/src/lib.rs

```diff
@@ -227,6 +235,61 @@ impl From<Seasonality> for augurs_prophet::Seasonality {
     }
 }
 
+/// Can be used to specify custom regressors
+#[derive(Clone, Debug, Default, Deserialize, Serialize, Tsify)]
+#[serde(rename_all = "camelCase")]
+#[tsify(from_wasm_abi, into_wasm_abi, type_prefix = "Prophet")]
+pub struct Regressor {
+    mode: FeatureMode,
```

Comment:
This could probably be optional too.

```suggestion
    /// The mode of this regressor.
    ///
    /// Defaults to "additive".
    #[serde(default)]
    mode: FeatureMode,
```